draft-ietf-httpbis-incremental.md

```diff
@@ -162,9 +162,9 @@ under which the intermediaries might reject requests.
 
 Some intermediaries inspect the payload of HTTP messages and forward them only
 if their content is deemed safe. Any feature that depends on seeing the
-entirety of the message in this way is incompatible with incremental delivery,
-so these intermediaries need to reject requests unless the entire message is
-received.
+entirety of the message in this way is incompatible with incremental delivery.
+An intermediary therefore MUST reject requests that ask for incremental forwarding
```

Comment:
Considering that incremental delivery itself is a SHOULD, use of MUST seems a bit odd to me.

Also, if we are to use an RFC 2119 term for permanent rejection, we might want to do the same for temporary rejection as well.

Reply:
Thank you in advance!

I'm not sure if it's perfectly related, but we had the discussion between MUST and SHOULD in https://github.com/httpwg/http-extensions/pull/3039.